packages/client/src/nfts/Nfts.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { Nfts } from '.';
+import { setupRandomAuthentication } from '../authentication/__helpers__/setupAuthentication';
+import { mumbaiSandbox } from '../consts/environments';
+
+const testConfig = {
+  environment: mumbaiSandbox,
+};
+
+describe(`Given the ${Nfts.name} configured to work with sandbox`, () => {
+  describe(`and the instance is not authenticated`, () => {
+    const nfts = new Nfts(testConfig);
+
+    describe(`when a method ${Nfts.prototype.fetch.name} is called`, () => {
```

Comment:
nit: in several places, maybe files.
```suggestion
    describe(`when the method ${Nfts.prototype.fetch.name} is called`, () => {
```